src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Common/Builder.cs

```diff
@@ -288,6 +292,14 @@ protected bool BuildInternal (string projectOrSolution, string target, string []
 			if (AutomaticNuGetRestore && restore && !UseDotNet) {
 				args.Append (" /restore");
 			}
+			if (MaxCpuCount != null) {
+				if (!string.Equals (Path.GetFileNameWithoutExtension (psi.FileName), "xabuild", StringComparison.OrdinalIgnoreCase)) {
+					args.Append ($" /maxCpuCount:{MaxCpuCount}");
+					args.Append (" /nodeReuse:false"); // Disable the MSBuild daemon
```

Comment:
I thought part of the problem was that nodes are reused? Won't this stop that from happening?

Reply:
When I didn't have this, `MSBuild.exe` would hang around and try to be reused. I thought it would cause file locking on Windows. I don't think `/nr:false` affects the current build.

If I use 16.7 this test fails with:
```
> msbuild .\bin\TestDebug\temp\SolutionBuildSeveralProjects\SolutionBuildSeveralProjects.sln /maxCpuCount:4 /nodeReuse:false
...
"C:\src\xamarin-android\bin\TestDebug\temp\SolutionBuildSeveralProjects\SolutionBuildSeveralProjects.sln" (default target) (1) ->
"C:\src\xamarin-android\bin\TestDebug\temp\SolutionBuildSeveralProjects\App1\App1.csproj" (default target) (2) ->
(CoreCompile target) ->
    Foo.cs(3,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name2'
    Foo.cs(4,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name3'
    Foo.cs(5,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name4'
    Foo.cs(7,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name6'
    Foo.cs(8,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name7'
    Foo.cs(9,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name8'
    Foo.cs(10,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name9'
"C:\src\xamarin-android\bin\TestDebug\temp\SolutionBuildSeveralProjects\SolutionBuildSeveralProjects.sln" (default target) (1) ->
"C:\src\xamarin-android\bin\TestDebug\temp\SolutionBuildSeveralProjects\App2\App2.csproj" (default target) (7) ->
    Foo.cs(3,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name2'
    Foo.cs(4,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name3'
    Foo.cs(5,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name4'
    Foo.cs(7,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name6'
    Foo.cs(8,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name7'
    Foo.cs(9,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name8'
    Foo.cs(10,42): error CS0117: 'Resource.String' does not contain a definition for 'library_name9'

0 Warning(s)
14 Error(s)
```